src/integration/pages/articles/imageCopyrightBBC.js

```diff
@@ -0,0 +1,12 @@
+export default () => {
+  describe('Image Copyright BBC', () => {
+    const copyrightTextEl = document.querySelector('main figure p');
+    it('should not be in the document', () => {
+      expect(copyrightTextEl).not.toBeInTheDocument();
+    });
```

Comment:
I think this test should only be checking for existence on the page
```suggestion
    it('should be in the document', () => {
      expect(copyrightTextEl).toBeInTheDocument();
    });
```

Reply:
This could also possibly explain the null snapshots above, so perhaps the Xpath is incorrect? `main figure p`?

Reply:
When there is BBC copyright, it isn't supposed to show on the page. So it needs to check that it isn't present. Now I see that query selecting for an element that isn't there though is of course giving us null... I'll rethink how to do this.

Reply:
Ahhhh OK, I had forgotten that we didn't display our own copyright 😄 . We can just remove the snapshot test